pkg/config/replica_config.go

```diff
@@ -49,6 +49,9 @@ var defaultReplicaConfig = &ReplicaConfig{
 		FlushIntervalInMs: 1000,
 		Storage:           "",
 	},
+	TableActor: &TableActorConfig{
```

Comment:
Could you add to Debug section for now? We can move it out when table actor is GA.

Reply:
ok